internal/expand/tree.go

```diff
@@ -85,3 +117,58 @@ func (t *Tree) UnmarshalJSON(v []byte) error {
 
 	return nil
 }
+
+func (t *Tree) ToGRPC() *acl.SubjectTree {
```

Comment:
I'd prefer naming all appearances in the code base of `ToGRPC` and `FromGRPC` to `ToProto`/`FromProto` since it's actually Protocol Buffer messages that are converted, not gRPC messages particularly. Would you rename this in another PR please?